src/interaction/SmoothSquareWell.py

```diff
@@ -0,0 +1,101 @@
+#  Copyright (C) 2017,2018
+#      Max Planck Institute for Polymer Research
+#
+#  This file is part of ESPResSo++.
+#
+#  ESPResSo++ is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  ESPResSo++ is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+r"""
+***************************************
+espressopp.integrator.SmoothSquareWell
+***************************************
+This is an inplementation of the smoothed square-well potential from Leitold and Dellago JCP 141 (2014).
```

Comment:
Please, put here the equation that this potential realized. Also, would be good to have url to the article instead of a weak citation.

e.g.

```
.. math::

   \frac{ \sum_{t=0}^{N}f(t,k) }{N}
```